deepface/DeepFace.py

```diff
@@ -299,6 +300,10 @@ def find(
         silent (boolean): Suppress or allow some log messages for a quieter analysis process
             (default is False).
 
+        refresh_data_base (boolean): Sincronizes the images representation (pkl) file with the 
```

Comment:
typo: refresh_data_base instead of refresh_database.

also it should be synchronizes instead of sincronizes